build.sbt

```diff
@@ -147,7 +147,24 @@ lazy val lambdaHttp4s = crossProject(JSPlatform, JVMPlatform)
     )
   )
   .settings(commonSettings)
-  .dependsOn(lambda % "compile->compile;test->test")
+  .dependsOn(lambda)
+
+lazy val cloudflareWorker = project
+  .in(file("cloudflare-worker"))
+  .enablePlugins(ScalaJSPlugin)
+  .settings(
+    name := "feral-cloudflare-worker",
+    libraryDependencies ++= Seq(
+      "org.typelevel" %%% "cats-effect" % catsEffectVersion,
+      "co.fs2" %%% "fs2-core" % fs2Version,
+      "io.circe" %%% "circe-generic" % circeVersion,
+      "io.circe" %%% "circe-scalajs" % circeVersion,
+      "org.http4s" %%% "http4s-client" % http4sVersion
+    )
+  ).dependsOn(core.js)
+  
+  .settings(commonSettings)
+  .dependsOn(sbtLambda)
```

Comment:
If it depends on Lambda I get a type error:

```
/scala/feral/build.sbt:167: error: type mismatch;
 found   : sbtcrossproject.CrossProject
 required: sbt.ClasspathDep[sbt.ProjectReference]
  .dependsOn(lambda)
```
it's `sbtLambda` because that's what made the type error go away :p

Reply:
It shouldn't depend on lambda either :) cloudflare and AWS lambda are totally different :P

Reply:
that makes a lot more sense, I wasn't thinking much when I was editing this build.sbt hehe.